tao-parser/src/test/resources/org/treeannotation/tao-html.tao

```diff
@@ -0,0 +1,133 @@
+[

```

Comment:
curiously this is both valid JSON and valid TAO at the same time :D
but it's more JSON, so not what we want
will get to this later

Reply:
ok!

Reply:
Created a PR to this PR with an update: https://github.com/tree-annotation/tao-parser-scala/pull/2